src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3584,3 +3650,309 @@ map<long, map<long, unordered_set<long>>> calculateEgoNet(string graphID, string
         }
     }
 }
+
+void calculateLocalPageRank(string graphID, string partitionID, int serverPort, int k,
```

Comment:
Rather than having `k`, can we use more descriptive variable name here?

Reply:
Fixed

Reply:
Ok